packages/esm-patient-orders-app/src/hooks/useOrderPrice.ts

```diff
@@ -0,0 +1,25 @@
+import { type OrderPriceData } from '../types/order';
+import { type FetchResponse, fhirBaseUrl, openmrsFetch } from '@openmrs/esm-framework';
+import useSWR from 'swr';
+import { useMemo } from 'react';
+import { useIsBackendModuleInstalled } from './useIsBackendModuleInstalled';
+
+export const useOrderPrice = (orderItemUuid: string) => {
+  const { isInstalled, isLoading: isCheckingModules } = useIsBackendModuleInstalled(['fhirproxy', 'billing']);
```

Comment:
```suggestion
  const { isInstalled, isLoading: isCheckingModules } = useAreBackendModuleInstalled(['fhirproxy', 'billing']);
```

Reply:
@ibacher we'd still need to make sure these are added to the `routes.json`'s `backendDependencies`, right?

Reply:
I had initially added them, @vasharma05 pointed out that since this is a major use-case for all user, it would not be good to always have a snackbar popup for them that some modules are missing

Reply:
@usamaidrsk I should clarify - I meant specifying them as `optionalBackendDependencies` in the route registries (see https://github.com/openmrs/openmrs-esm-core/pull/1015). `optionalBackendDependencies` don't shouldn't get reported as missing - so that should not be a concern.

Reply:
Got it now @denniskigen ,
Thanks for clarifying, let me update that.

Reply:
I have added that @denniskigen

Reply:
Via an existing commit? I don't see it in the diff...

Reply:
The push didn't go through, I have just updated it now.